changelog.dd

```diff
@@ -73,6 +82,84 @@ assert(m.front[1] == "12");
 -------
 )
 
+$(LI $(LNAME2 optimization, `findLocalMin` was added to `std.numeric`.)
+    $(P $(REF findLocalMin, std,numeric) finds a real minimum of a real function `f(x)` via bracketing.)
+-------
+import std.numeric, std.math;
+
+auto ret = findLocalMin((double x) => (x-4)^^2, -1e7, 1e7);
+
+assert(ret.x.approxEqual(4.0));
+assert(ret.y.approxEqual(0.0));
+-------
+)
+
+$(LI $(LNAME2 slice_ptr, `ptr` property and public constructor were added to
+    `std.experimental.ndslice.slice.Slice`.)
+    $(P `ptr` property allows to access `Slice`'s
+        underlaying pointer or range. It is suggested to read about
```

Comment:
> It is suggested to read about

This is somewhat awkward English. Something like "Please refer to `$(REF Slice, std,experimental,ndslice,slice)`'s internal binary representation before using the property." would sound more natural.